crates/pixi_manifest/src/manifests/mod.rs

```diff
@@ -106,4 +108,40 @@ dummy = "3.11.*"
 
         assert!(dep.is_some());
     }
+
+    #[test]
+    fn test_get_or_insert_inline_table() {
```

Comment:
Some ideas:
- Test multiple levels of table depth
- Test if existing entries persist
- Test what happens when you mix the inline and normal table for the same information. 
I expect all of this to work so might not be to useful :man_shrugging:

Reply:
> Test if existing entries persist

That one made a lot of sense to me, so I added it.
I didn't implement the others :)